app/src/main/java/com/xmartlabs/scasas/criminalintent/model/CrimeLab.java

```diff
@@ -0,0 +1,50 @@
+package com.xmartlabs.scasas.criminalintent.model;
+
+import android.content.Context;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by scasas on 2/7/17.
+ */
+public class CrimeLab {
```

Comment:
rename to `CrimeController`, and move to a new controller package (it's not a model)